apps/meteor/tests/e2e/omnichannel/omnichannel-chat-history.spec.ts

```diff
@@ -72,5 +74,17 @@ test.describe('Omnichannel chat history', () => {
 			await agent.poHomeOmnichannel.contacts.contactInfo.historyItem.click();
 			await expect(agent.poHomeOmnichannel.contacts.contactInfo.historyMessage).toBeVisible();
 		});
+
+		await api.post('/permissions.update', { permissions: [{ _id: 'view-c-room', roles: [] }] });
+
+		await test.step('Expect to not be able to see conversation history', async () => {
+			await agent.page.reload();
+
+			await agent.poHomeOmnichannel.contacts.contactInfo.historyItem.click();
+			await agent.poHomeOmnichannel.contacts.contactInfo.historyMessage.click();
+
+			// Should not show the NoSubscribedRoom.tsx component on livechat rooms
+			await expect(agent.page.getByRole('button', { name: 'Join_channel' })).not.toBeVisible();
```

Comment:
You should assert what is actually visible, instead of what should not be visible.